./tests/regressiontests/test_client_regress/models.py:26: warning: test_contains has 81 NLOC,1
./tests/regressiontests/templates/tests.py:377: warning: test_templates has 87 NLOC,2
./tests/runtests.py:153: warning: bisect_tests has 47 NLOC,1
./django/templatetags/i18n.py:282: warning: do_block_translate has 49 NLOC,1
./django/middleware/common.py:35: warning: process_request has 41 NLOC,1
./django/forms/models.py:132: warning: fields_for_model has 32 NLOC,3
./django/forms/forms.py:138: warning: _html_output has 52 NLOC,1
./django/core/management/commands/inspectdb.py:27: warning: handle_inspection has 77 NLOC,1
./django/core/management/commands/dumpdata.py:30: warning: handle has 77 NLOC,2
./django/core/management/commands/loaddata.py:31: warning: handle has 163 NLOC,1
./django/core/management/commands/syncdb.py:23: warning: handle_noargs has 98 NLOC,1
./django/core/management/__init__.py:264: warning: autocomplete has 37 NLOC,1
./django/core/management/validation.py:21: warning: get_validation_errors has 239 NLOC,1
./django/core/handlers/base.py:72: warning: get_response has 85 NLOC,1
./django/core/servers/fastcgi.py:89: warning: runfastcgi has 77 NLOC,1
./django/test/_doctest.py:905: warning: _find has 48 NLOC,2
./django/template/defaulttags.py:19: warning: token_kwargs has 29 NLOC,1
./django/utils/unittest/loader.py:97: warning: loadTestsFromName has 37 NLOC,1
./django/utils/unittest/case.py:306: warning: run has 69 NLOC,1
./django/utils/unittest/main.py:114: warning: parseArgs has 35 NLOC,1
./django/utils/translation/trans_real.py:360: warning: get_language_from_request has 32 NLOC,2
./django/utils/simplejson/decoder.py:57: warning: py_scanstring has 59 NLOC,2
./django/utils/simplejson/scanner.py:27: warning: py_make_scanner._scan_once has 33 NLOC,1
./django/utils/simplejson/encoder.py:322: warning: _iterencode_dict has 74 NLOC,2
./django/utils/numberformat.py:5: warning: format has 29 NLOC,1
./django/utils/html.py:102: warning: urlize has 38 NLOC,1
./django/utils/regex_helper.py:42: warning: normalize has 81 NLOC,2
./django/utils/dictconfig.py:272: warning: configure has 108 NLOC,2
./django/utils/module_loading.py:6: warning: module_has_submodule has 45 NLOC,1
./django/contrib/comments/views/comments.py:28: warning: post_comment has 71 NLOC,1
./django/contrib/localflavor/id/forms.py:110: warning: clean has 32 NLOC,1
./django/contrib/auth/management/commands/createsuperuser.py:39: warning: handle has 76 NLOC,1
./django/contrib/admin/templatetags/admin_list.py:128: warning: items_for_result has 62 NLOC,2
./django/contrib/admin/views/main.py:178: warning: get_query_set has 69 NLOC,1
./django/contrib/admin/media/js/SelectFilter2.js:111: warning: filter_key_down has 16 NLOC,1
./django/contrib/admin/media/js/getElementsBySelector.js:25: warning: document.getElementsBySelector has 121 NLOC,1
./django/contrib/admin/media/js/jquery.js:75: warning: init has 67 NLOC,20
./django/contrib/admin/options.py:207: warning: lookup_allowed has 31 NLOC,4
./django/contrib/admin/validation.py:13: warning: validate has 110 NLOC,2
./django/contrib/staticfiles/management/commands/collectstatic.py:51: warning: handle_noargs has 47 NLOC,1
./django/contrib/contenttypes/views.py:6: warning: shortcut has 45 NLOC,1
./django/contrib/gis/utils/layermapping.py:155: warning: check_layer has 54 NLOC,2
./django/contrib/gis/db/backends/postgis/operations.py:483: warning: spatial_lookup_sql has 46 NLOC,1
./django/contrib/gis/db/models/sql/compiler.py:11: warning: get_columns has 66 NLOC,1
./django/contrib/gis/db/models/query.py:571: warning: _distance_attribute has 85 NLOC,1
./django/contrib/gis/templates/gis/google/google-map.js:0: warning: *global* has 35 NLOC,8
./django/contrib/admindocs/views.py:186: warning: model_detail has 80 NLOC,1
./django/http/multipartparser.py:95: warning: parse has 98 NLOC,1
./django/db/backends/oracle/creation.py:45: warning: _create_test_db has 61 NLOC,1
./django/db/backends/oracle/base.py:145: warning: convert_values has 27 NLOC,1
./django/db/backends/creation.py:29: warning: sql_create_model has 52 NLOC,1
./django/db/models/fields/related.py:319: warning: __set__ has 33 NLOC,1
./django/db/models/fields/__init__.py:303: warning: get_db_prep_lookup has 35 NLOC,1
./django/db/models/sql/compiler.py:47: warning: as_sql has 50 NLOC,3
./django/db/models/sql/query.py:426: warning: combine has 68 NLOC,4
./django/db/models/sql/where.py:75: warning: as_sql has 41 NLOC,2
./django/db/models/query.py:148: warning: __getitem__ has 37 NLOC,3
./django/db/models/deletion.py:216: warning: delete has 36 NLOC,
./django/db/models/base.py:30: warning: __new__ has 118 NLOC,3
./django/views/decorators/http.py:78: warning: condition.condition.decorator.inner has 62 NLOC,1
./django/views/debug.py:82: warning: get_traceback_html has 64 NLOC,1
./django/views/i18n.py:170: warning: javascript_catalog has 89 NLOC,1
./django/conf/__init__.py:80: warning: __init__ has 42 NLOC,1